tests/js/blockPRNG.scrypttest.js

```diff
@@ -0,0 +1,185 @@
+const { expect } = require('chai');
+const { bsv, buildContractClass, PubKey, toHex, Sha256, Bytes, getPreimage, signTx, buildTypeClasses, num2bin } = require('scryptlib');
+const { num2hex, sha256d, reverseEndian, compileContract, inputIndex, inputSatoshis, uint322bin, toLittleIndian, pdiff2Target } = require('../../helper');
+const BN = bsv.crypto.BN
+
+function newTx() {
```

Comment:
Why is this shared code not in helper?

Reply:
move to blockchainhelper, helper.js have a `newTx`, which use random txid